multilinestring.go

```diff
@@ -25,6 +25,19 @@ func (g *MultiLineString) Area() float64 {
 	return 0
 }
 
+// Empty returns whether the MultiLineString is empty, or contains entirely empty LineStrings.
+func (g *MultiLineString) Empty() bool {
+	if g.geom2.Empty() {
+		return true
+	}
+	for i := 0; i < g.NumLineStrings(); i++ {
+		if !g.LineString(i).Empty() {
+			return false
+		}
+	}
+	return true
+}
```

Comment:
If we remove this function, i.e. `MultiLineString.Empty` uses `geom2.Empty` only, then the tests still pass. Is there an example `MultiLineString` where `geom2.Empty` fails and the loop over the individual `LineString`s is needed?

Reply:
oh nice! i've worked backwards to this fix haha (i added this function, tested it to get the panics below, didn't realise it wasn't needed at all - testing saves lives!).